lib/reline/key_actor/emacs.rb

```diff
@@ -55,7 +55,7 @@ class Reline::KeyActor::Emacs < Reline::KeyActor::Base
     #  26 ^Z
     :ed_ignore,
     #  27 ^[
-    :ed_unassigned,
+    :redo,
```

Comment:
If you have any opinions about which key bindings should redo be assigned to, please let me know.